test/vcf/debugulator_integration_test.cpp

```diff
@@ -74,7 +74,7 @@ namespace ebi
 
       boost::filesystem::remove(fixed_vcf_path);
 
-      if (not successfully_fixed) {
+      if (! successfully_fixed) {
```

Comment:
Remove unnecessary whitespace, it was fine with "not" because it was 2 word, but not with the exclamation mark. It might apply to other files.

Reply:
It was only in this file. I will remove it.